src/main/java/mekanism/common/content/entangloporter/InventoryFrequency.java

```diff
@@ -21,8 +21,8 @@
 
     public static final String ENTANGLOPORTER = "Entangloporter";
 
-    public static final int FLUID_TANK_SIZE = Tier.FluidTankTier.ULTIMATE.output;
-    public static final int GAS_TANK_SIZE = Tier.GasTankTier.ULTIMATE.output;
+    public static final int FLUID_TANK_SIZE = Tier.FluidTankTier.ULTIMATE.getOutput();
```

Comment:
Looks like these constants should be removed and their usages replaced by the config read